tests/suites/test_suite_x509write.function

```diff
@@ -28,6 +32,74 @@ size_t mbedtls_rsa_key_len_func( void *ctx )
 }
 #endif /* MBEDTLS_RSA_C */
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+static int wrap_key_as_opaque( mbedtls_pk_context *pk, psa_algorithm_t hash_alg )
+{
+    psa_key_slot_t key;
+    const mbedtls_ecp_keypair *ec;
+    psa_ecc_curve_t curve_id;
+    psa_key_type_t key_type;
+    psa_key_policy_t policy;
+    int key_len;
+    const int buf_len = 30 + 2 * MBEDTLS_ECP_MAX_BYTES; // Equivalent of ECP_PUB_DER_MAX_BYTES
+    unsigned char buf[buf_len];
+
+     /* Export the private key material in the format PSA wants.
+      * So far only ECDSA is supported. */
+    if( mbedtls_pk_get_type( pk ) != MBEDTLS_PK_ECKEY )
+        return( MBEDTLS_ERR_PK_TYPE_MISMATCH );
+
+    ec = mbedtls_pk_ec( *pk );
+    if( ( key_len = mbedtls_pk_write_key_der( pk, buf, buf_len ) ) <= 0 )
```

Comment:
If `psa_import_key()` work after that, it means you're using the old version of that API. I'l ask Hanno to pick up my version, then you can rebase over the utilities branch and us the version from there.